### PR TITLE
Several fixes and style changes

### PR DESCRIPTION
**Pull request by CMR248**
Several fixes in links and spelling of KB and QS. Style changes in KB. 
@Martin-Kuhl @Brilator

### CODE REVIEW

File: src/content/docs/git.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ The following table gives an overview about DataPLANT tools and services related
...
-**[ARC][KB-ARC]**  <br> (Annotated Research Context) | Standard | <ul><li>ARCs are git repositories</li><li>Package data with metadata in a defined format</li></ul>
-**[DataHUB][DataHUB]** | Service | <ul><li>DataPLANT-customized GitLab instance</li><li>Infrastructure-as-code: on-premise solution</li><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
+**ArcCommander** | Tool | **Collect, structure and share:** <ul><li>Add bibliographical metadata to your ARC</li><li>ARC version control and sharing via DataPLANT's DataHUB</li><li>Automated metadata referencing and version control as your ARC grows</li></ul>
+**[ARC][KB-ARC]**  <br> (Annotated Research Context) | Standard | **Structure:** <ul><li>ARCs are git repositories</li><li>Package data with metadata in a defined format</li></ul>
+**[DataHUB](datahub.html)** | Service | **Share:** <ul><li>DataPLANT-customized GitLab instance</li><li>Infrastructure-as-code: on-premise solution</li><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
```
Comment (Freymaurer) [inline src/content/docs/git.md:42]: Maybe add a cross reference here, so if a link gets changed we only need to change it in one place.

```
[KB-Datahub]: ./datahub.html
```

File: src/content/docs/quickstart_arc.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ todo:
 
 We are very happy that you chose our tools and infrastructure to create and share your own ARCs. In this QuickStart we focus on how to use the "ARC Commander" to store your data and "SWATE" to enrich it with metadata.
 
-This document is work in progress. If you experience any inconsistencies, have questions or would like to suggest additions, please feel free to send a message to: <a href="mailto:info@nfdi4plants.org?subject=ARC%20QuickStart">info[at]nfdi4plants.org</a> or open in issue at <https://github.com/nfdi4plants/quickstart>.
+This document is work in progress. If you experience any inconsistencies, have questions or would like to suggest additions, please feel free to [contact us](mailto:info@nfdi4plants.org?subject=ARC%20QuickStart) or open in issue in our [Helpdesk](<https://github.com/nfdi4plants/quickstart>).
```
Comment (Freymaurer) [inline src/content/docs/quickstart_arc.md:18]: Please obfuscate the email. Not sure how to do this out of the box in markdown.

File: src/content/docs/repositories.md [hunk 3]
```diff
@@ -69,9 +69,9 @@ The following table gives an overview about DataPLANT tools and services related
...
-**[ArcCommander][KB-arccommander]** | Tool | **Collect, structure and share:** <ul><li>Add bibliographical metadata to your ARC</li><li>ARC version control and sharing via DataPLANT's DataHUB</li><li>Automated metadata referencing and version control as your ARC grows</li></ul>
-**[DataHUB][DataHUB]** | Service | **Share:** <ul><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
+**Swate** <br> (Swate Workflow Annotation Tool for Excel) | Tool | **Collect and structure:** <ul><li>Annotate experimental and computational workflows with ISA metadata schema</li><li>Easy use of ontologies and controlled vocabularies</li><li>Metadata templates for versatile data types</li></ul>
+**ArcCommander** | Tool | **Collect, structure and share:** <ul><li>Add bibliographical metadata to your ARC</li><li>ARC version control and sharing via DataPLANT's DataHUB</li><li>Automated metadata referencing and version control as your ARC grows</li></ul>
+**[DataHUB](datahub.html)** | Service | **Share:** <ul><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
```
Comment (Freymaurer) [inline src/content/docs/repositories.md:74]: Again, might be better to use `[KB-Datahub]: ./datahub.html` for easier maintenance.

**Review by Freymaurer — CHANGES_REQUESTED** [review]
Changes are of course open for discussion.

File: src/content/docs/repositories.md (hunk 3 above)
Comment (Martin-Kuhl) [reply src/content/docs/repositories.md:74]: Yes.. I think a uniform solution is the way to go.

File: src/content/docs/git.md (hunk 1 above)
Comment (Martin-Kuhl) [reply src/content/docs/git.md:42]: Should have been fixed in [#77](https://github.com/nfdi4plants/nfdi4plants.github.io/pull/77/commits). Not sure why this one was closed...

**Pull request merged**